### PR TITLE
Updated CHAdeMO module with current and voltage checks.

### PR DESCRIPTION
**Pull request by jsphuebner**
I see it is not used yet but I added some sanity checking to the CHAdeMO module

### CODE REVIEW

File: src/chademo.cpp [hunk 1]
```diff
@@ -64,23 +66,56 @@ void ChaDeMo::SetChargeCurrent(uint8_t current)
...
-void ChaDeMo::SendMessages()
+void ChaDeMo::CheckSensorDeviation(uint16_t internalVoltage)
+{
+   int vtgDev = (int)internalVoltage - (int)chargerOutputVoltage;
+
+   vtgDev = ABS(vtgDev);
+
+   if (vtgDev > 10)
+   {
+      vtgTimeout++;
```
Comment (mdrobnak) [inline src/chademo.cpp:77]: I'd suggest that a better name might be vtgErrCnt as this isn't a timeout. Same for curTimeout.

**Comment by mdrobnak** [comment]
Aside from the variable naming, looks good.

File: src/chademo.cpp (hunk 1 above)
Comment (jsphuebner) [reply src/chademo.cpp:77]: It's both. The function must be called at 100ms intervals so a counter of 10 corresponds to 1000 ms. The chademo spec specifies a timeout. So I thought that's what I'd call it.

Comment (mdrobnak) [reply src/chademo.cpp:77]: Ok, makes sense then. Thanks for the additional context.

**Pull request merged**